tests/Tests/600__ai_apps/660__openvino/660__openvino.robot

```diff
@@ -26,19 +26,33 @@ Verify OpenVino Is Available In RHODS Dashboard Explore Page
 Verify Openvino Operator Can Be Installed Using OpenShift Console
    [Tags]   Tier2
    ...      ODS-675
+   ...      ODS-652
    ...      ODS-495
    ...      ODS-1236
-   [Documentation]  This Test Case Installed Openvino operator in Openshift cluster
+   [Documentation]  This Test Case Installed Openvino operator in Openshift cluster,
+   ...              Cheks whether RHODS notification shows the Notebook images are running after installing openvino
    ...               and Check and Launch AIKIT notebook image from RHODS dashboard
    Check And Install Operator in Openshift    ${openvino_operator_name}   ${openvino_appname}
    Create Tabname Instance For Installed Operator        ${openvino_operator_name}       Notebook    redhat-ods-applications
    Wait Until Keyword Succeeds    900  1     Check Image Build Status   Complete     openvino-notebook
+   Get Build Status    namespace=redhat-ods-applications    build_search_term=openvino-notebook
```

Comment:
how is this line used?

Reply:
This lines navigates to the builds and searches the operator name

Reply:
but you don't need to run all the lines in "Get Build Status" and moreover the status is not used here. You could just use `Navigate To Page    Builds    Builds` + `Select Project By Name`.